spec/requests/invites_controller_spec.rb

```diff
@@ -91,6 +91,7 @@
 
       it "fails if you can't invite to the forum" do
         sign_in(Fabricate(:user))
+        SiteSetting.min_trust_to_allow_pm_invite = 2
```

Comment:
I don't think this is correct because `Guardian#can_invite_to_forum?` uses the `SiteSetting.min_trust_to_allow_invite`. I'm also not sure if we need to set the site setting when the default value is already 2.